backend/migrations/versions/5d3bf271b78d_update_uq_score_for_guestlist.py

```diff
@@ -0,0 +1,517 @@
+"""update uq score for guestlist
+
+Revision ID: 5d3bf271b78d
+Revises: 19f6b4f5ce2e
+Create Date: 2024-07-19 14:11:00.850135
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from eth_utils import to_checksum_address
+
+revision = "5d3bf271b78d"
+down_revision = "19f6b4f5ce2e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Guest-list is a list of stings that represent user addresses
+    # For all users in the Guest-list, if they have a UQ score, update it to 1.0 only for epoch 4
+    query = f"""
+        UPDATE uniqueness_quotients
+        SET score = '1.0'
+        WHERE score = '0.2' AND epoch = 4 AND user_id IN (
+            SELECT id FROM users WHERE address IN (
+                {", ".join([f"'{address}'" for address in user_addresses])}
+            )       
+        );
+    """
+    op.execute(query)
+
+
+def downgrade():
+    query = f"""
+        UPDATE uniqueness_quotients
```

Comment:
the possible issue here is that we'll downgrade every person from the guestlist (not only ppl from the upgrade) but it's not very likely to execute this so not a blocker